Mathlib/Control/Basic.lean

```diff
@@ -0,0 +1,17 @@
+/-
+Copyright (c) 2022 E.W.Ayers. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: E.W.Ayers
+-/
+
+/-!
+
+# Basic instances and functions for monads and other structures.
+
+-/
+
+instance : Monad id where
```

Comment:
BTW, there's `Id` for the type which has a monad instance.

Reply:
ahhhhh